hardware/asic/skywater/Makefile

```diff
@@ -1,46 +1,35 @@
 ROOT_DIR=../../..
-
+include ../asic.mk
 OPENLANE_DESIGNS:=$(OPENLANE_HOME)/designs
-IMAGE_NAME ?= efabless/openlane:2021.07.29_04.49.46 #latest
+IMAGE_NAME ?= efabless/openlane:2021.07.29_04.49.46
 ASIC_NODE=skywater
-#defmacro := -D
+SRAM_W = 7
 MKDIR := mkdir -p
-include ../asic.mk
-
-VSRC+=iob_dp_ram.v
-
-# Memory technology
-ASIC_MEM_TECH:=skywater130
-
-# Memory type
-ASIC_DPRAM_TYPE:=spram
-
+VSRC+=iob_sp_ram_be.v
```

Comment:
are these the wrappers? what have you done with the others that were in VSRC?

Reply:
this is the wrapper that is generated by the Makefile by calling memakerwrap.py

Reply:
I think with asic=1 in asic.mk we do not get wrappers in VSRC and they are generated and added by skywater Makefile